src/type-translator.ts

```diff
@@ -503,13 +504,15 @@ export class TypeTranslator {
   /** @return true if sym should always have type {?}. */
   isBlackListed(symbol: ts.Symbol): boolean {
     if (this.pathBlackList === undefined) return false;
-    const pathBlackList = this.pathBlackList;
     if (symbol.declarations === undefined) {
       this.warn('symbol has no declarations');
       return true;
     }
+    // Normalize paths to not break checks on Windows.
+    const pathBlackList = new Set<string>(
```

Comment:
can we arrange the code such that we only do this once, e.g. in the constructor, instead of on every check? This method is called frequently, doing the work over and over again will likely be a performance issue.